### ISSUE TITLE
Sample request: Add/remove a host's vmnics from a VDS

### ISSUE THREAD

**Issue by EB123** [vmware/pyvmomi-community-samples#170]
Hi all,

I trying to write something that will be able to add/remove a specific host's vmnics from a specific VDS.

Basically what i'm trying tp simulate is (how it's done from the client): 
Select host -> Configuration -> Networking -> vsphere distributed switch -> select "manage physical adapter" for the desired VDS.

From my understanding I need to use " UpdateNetworkConfig" method on a vim.host.NetworkSystem object with a config spec where everything is "unset" besides the HostProxySwitch.Config, but i'm not exaclty sure what keys i must fill myself and how do i remove a vmnic when the config spec mode is set to "modify".

Basically i'm trying to do something pretty similar to this issue - https://github.com/vmware/pyvmomi-community-samples/issues/119 , only i want to do it by reconfiguring the host and not the VDS (i also dont need to add already-used vmnics, so that's not an issue).

Thanks for your help.

**Comment by prziborowski** [comment]
Is there a reason you want to use UpdateNetworkConfig from networkSystem rather than doing a Reconfigure on the vDS (similar to #119)?
Basically if you don't have to deal with busy vmnics, then what #119 has should work. To remove, use `dvs_host_config.operation = vim.ConfigSpecOperation.remove`

**Comment by plrnr** [comment]
vim.ConfigSpecOperation.remove removes the specified hypervisor from a DVS. Is there a way to just remove a specific vmnic?
